pages/docs/why-effect.mdx

```diff
@@ -0,0 +1,71 @@
+# Why Effect?
+
+## Motivation
+
+Programming is challenging. When we build libraries and apps, we look to many tools to handle the complexity and make our day-to-day more manageable.
+
+One of the most important tools we have in TypeScript is the compiler. It is the first line of defense against bugs, domain errors, and general complexity. Effect presents a new way of thinking about programming in TypeScript.
```

Comment:
I would suggest moving the statements about the TypeScript compiler down below in the paragraph that starts with `In "typical" TypeScript`. And then the final sentence of this paragraph can become the last sentence of the first paragraph.

Reply:
Done, thanks!